docs/changelog.rst

```diff
@@ -123,6 +129,14 @@ Cleanups
 - Catch errors when loading :confval:`logging/config_file`.
   (Fixes: :issue:`1320`)
 
+Audio
+-----
+
+- **Breaking:** The audio scanner now returns ISO-8601 formatted strings
```

Comment:
@tkem this one is probably relevant for you with respect to mopidy-local-sqlite. Other uses of the scanner is mostly tunein and the likes, @kingosticks I presume this won't effect that code?